geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -176,8 +176,18 @@
   private static final int prMetaDataSentCountId;
 
   private static final int localMaxMemoryId;
+  static final int bucketClearsId;
+  static final int bucketClearTimeId;
+  static final int bucketClearsInProgressId;
 
   static {
+    final String bucketClearsDesc =
+        "The total number of times a clear has been done on this region and it's bucket regions";
```

Comment:
I think this is better: "The total number of times a bucket of this partitioned region has been cleared."

Reply:
done